astropy/io/misc/tests/test_parquet.py

```diff
@@ -0,0 +1,638 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import pytest
+import numpy as np
+
+from astropy.table import Table, QTable, NdarrayMixin, Column
+from astropy.table.table_helpers import simple_table
+
+from astropy import units as u
+
+from astropy.coordinates import (SkyCoord, Latitude, Longitude, Angle, EarthLocation,
+                                 SphericalRepresentation, CartesianRepresentation,
+                                 SphericalCosLatDifferential)
+from astropy.time import Time, TimeDelta
+from astropy.units import allclose as quantity_allclose
+from astropy.units.quantity import QuantityInfo
+from astropy.utils.exceptions import AstropyUserWarning
+from astropy.utils.data import get_pkg_data_filename
+from astropy.utils.compat.optional_deps import HAS_PYARROW  # noqa
+if HAS_PYARROW:
+    import pyarrow
```

Comment:
It looks like pyarrow is necessary for all these tests, so its probably easier to use ``importskip``. https://docs.pytest.org/en/6.2.x/reference.html#pytest-importorskip 
than all the ``@pytest.mark.skipif('not HAS_PYARROW')``